pkg/plugin/test-keeper/plugin/comment_message.go

```diff
@@ -29,18 +29,22 @@ const (
 		"but the plugin wasn't able to retrieve it from the defined location (%s). Make sure it is either a valid URL or a " +
 		"path to an existing file in this repository."
 
+	sadIke = `<p align="center">
+  				<img src="http://design.jboss.org/arquillian/logo/ui/images/failure/arquillian_ui_failure_256px.svg">
+			  </p>`
```

Comment:
the image is too big to my taste - it doesn't bring any additional value to the comment, but occupy 50% of it. I'm not against to having it there, but I would make it either smaller or make the text flowed around the image:
![obrazek](https://user-images.githubusercontent.com/1510709/37908404-6b17f2b2-3108-11e8-888e-310898f943cc.png)
the code is `<img align="right" src="http://design.jboss.org/arquillian/logo/ui/images/failure/arquillian_ui_failure_256px.png">`

Reply:
Sure, let's make it smaller